proxy.go

```diff
@@ -23,8 +23,10 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-// tmpDir temporary path to store ongoing queries results
-const tmpDir = "/tmp"
+const (
+	tmpDir             = "/tmp"                // tmpDir temporary path to store ongoing queries results
+	defaultHostCounter = uint32((1 << 32) - 1) // default value for Host.Counter; Short-term fix issue #322
```

Comment:
be-careful, with such a big value you'll likely overflow. 

You should put 1 << 31

Reply:
I'll test 1<<31
However, when it's 1<<32 `s.host.inc()` will be called https://github.com/ContentSquare/chproxy/blob/5872a429dd50e8c0a37646e4147eb1fcff217a9e/proxy.go#L100
as result value will be 0

Reply:
yes, and if 2 queries happens at the same time the value will be 2^32